tests/js/instrumentedEnv/userEventIntegration.ts

```diff
@@ -0,0 +1,51 @@
+import {Hub} from '@sentry/hub';
+import {fill, isThenable, loadModule} from '@sentry/utils';
+
+const moduleName = '@testing-library/user-event';
```

Comment:
why in a variable like this?

Reply:
no particular reason, an artifact of all the ways i tried to do the monkey patching. i can inline it